src/pint/toa.py

```diff
@@ -1708,6 +1711,11 @@ def write_TOA_file(self, filename, name="unk", format="tempo2"):
         if format.upper() in ("TEMPO2", "1"):
             outf.write("FORMAT 1\n")
 
+        # Warn the user if any of the TOAs have -ignore or -cut flags that
+        # they will be commented in the output file
+        if isinstance(commentflag, str) and len(self.get_flag_value(commentflag)[1]):
+            log.warning(f"TOAs with '-{commentflag}' flags will be commented")
```

Comment:
We already have too many messages. If you are worried about the user being surprised by commented TOAs, make the default None; if you are not worried, leave the warning out.

Reply:
I'm fine leaving the warning out.  I put it in there specifically because you said that you were worried that people might be surprised by having TOAs commented.

Reply:
Indeed; my preferred solution is to default to not commenting any TOAs. But I don't think a warning is a good idea.